site/src/docs/components/login/usage.mdx

```diff
@@ -0,0 +1,622 @@
+  - <InternalLink href="/components/login/usage#withauthenticateduser">WithAuthenticatedUser</InternalLink>
+  - <InternalLink href="/components/login/usage#withoutauthenticateduser">WithoutAuthenticatedUser</InternalLink>
+
+- <InternalLink href="/components/login/usage#hooks">Hooks</InternalLink>
+
+  - <InternalLink href="/components/login/usage#useoidcclient">useOidcClient</InternalLink>
+  - <InternalLink href="/components/login/usage#useauthenticateduser">useAuthenticatedUser</InternalLink>
+  - <InternalLink href="/components/login/usage#usecachedamr">useCachedAmr</InternalLink>
+  - <InternalLink href="/components/login/usage#useoidcclienttracking">useOidcClientTracking</InternalLink>
+  - <InternalLink href="/components/login/usage#useapitokensclient">useApiTokensClient</InternalLink>
+  - <InternalLink href="/components/login/usage#useapitokens">useApiTokens</InternalLink>
+  - <InternalLink href="/components/login/usage#useapitokensclienttracking">useApiTokensClientTracking</InternalLink>
+  - <InternalLink href="/components/login/usage#usesessionpoller">useSessionPoller</InternalLink>
+  - <InternalLink href="/components/login/usage#usesessionpollertracking">useSessionPollerTracking</InternalLink>
+  - <InternalLink href="/components/login/usage#usesignallistener">useSignalListener</InternalLink>
+  - <InternalLink href="/components/login/usage#usesignaltrackingwithreturnvalue">useSignalTrackingWithReturnValue</InternalLink>
+  - <InternalLink href="/components/login/usage#usesignaltrackingwithcallback">useSignalTrackingWithCallback</InternalLink>
+
+- <InternalLink href="/components/login/usage#modules">Modules</InternalLink>
+
+  - <InternalLink href="/components/login/usage#oidc-client">Oidc Client</InternalLink>
+  - <InternalLink href="/components/login/usage#api-tokens-client">Api tokens client</InternalLink>
+  - <InternalLink href="/components/login/usage#session-poller">Session poller</InternalLink>
+
+- <InternalLink href="/components/login/usage#silent-renewal">Silent renewal</InternalLink>
+- <InternalLink href="/components/login/usage#signals">Signals</InternalLink>
+
+### Components
+
+HDS login components cover most use cases and also handle errors when logging in or getting tokens.
+
+Detailed information about all properties of each component is listed on the <InternalLink href="/components/login/api#components">API page</InternalLink>.
+
+#### Important
+
+All components and hooks require a <InternalLink href="/components/login/usage#loginprovider">LoginProvider</InternalLink> to provide context for them.
+
+#### LoginProvider
+
+This component creates a React context and initialises all modules. All components and hooks are required to render inside the LoginProvider or React Context will throw an error.
+Position the context in the component tree like any other React context. The context does not change and cause re-renders often, so it can be at the top level.
```

Comment:
Typo: "The context does not change **or** cause re-renders often,--".

Reply:
Hmmmm... It cannot cause a re-render without changing, so "and" is correct I think.